# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/app/profile/firefox.js
# Commit: ccd96b99aea2
# Full Hash: ccd96b99aea28e8214e585b864d3fb2245bac142
# Author: Atila Butkovits <abutkovits@mozilla.com>
# Date: 2025-06-07 08:31:23
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing failures complaining about fedcm-no-registered-idps.https.html.
#   
#   This reverts commit ba33457e0cbac24a8b7d963c45138c62e92a74ae.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r 4bfadfc33207 -r ccd96b99aea2 browser/app/profile/firefox.js
--- a/browser/app/profile/firefox.js	Fri Jun 06 21:07:41 2025 +0300
+++ b/browser/app/profile/firefox.js	Fri Jun 06 21:35:18 2025 +0300
@@ -3268,6 +3268,7 @@
 
 #ifdef NIGHTLY_BUILD
   pref("dom.security.credentialmanagement.identity.enabled", true);
+  pref("dom.security.credentialmanagement.identity.heavyweight.enabled", true);
 #endif
 
 pref("ui.new-webcompat-reporter.enabled", true);